expression/integration_test.go

```diff
@@ -5066,7 +5066,7 @@ func (s *testIntegrationSuite) TestTiDBInternalFunc(c *C) {
 	result.Check(testkit.Rows(rs))
 
 	// https://github.com/pingcap/tidb/issues/27434.
-	hexKey = "7480000000000000375F69800000000000000103800000000001D4C1023B6458"
```

Comment:
`select tidb_decode_key( '7480000000000000375F69800000000000000103800000000001D4C1023B6458' )` panics after adding the system table. `74` corresponds to `t` and `8000000000000037` corresponds to table id `55`. In the test, the table id of  the new added table `mysql.column_stats_usage` is exactly `55` so the panic happened. So I changed the table id from `8000000000000037` to `8000000000000FFF`. PTAL @tangenta